content/07.supplemental.md

```diff
@@ -2,12 +2,12 @@
 <!-- TODO: Add legends, update S4 if we keep immune deconv figures. -->
 
 
-![Figure S1. OpenPBTA Project Workflow, Related to Figure 1.](images/supplement/Figure-S1.png){#fig:FigS1}
+![**Figure S1. OpenPBTA Project Workflow, Related to Figure 1.** Biospecimens and data were collected by CBTN and PNOC and genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). Analyses in the green boxes were performed by contributors of the OpenPBTA project. Output files are denoted in blue. Figure created with biorender.com.](images/supplement/Figure-S1.png){#fig:FigS1}
```

Comment:
OSPL:

```suggestion
![**Figure S1. OpenPBTA Project Workflow, Related to Figure 1.** Biospecimens and data were collected by CBTN and PNOC.
Genomic sequencing and harmonization (orange boxes) were performed by the Kids First Data Resource Center (KFDRC). 
Analyses in the green boxes were performed by contributors of the OpenPBTA project.
Output files are denoted in blue. Figure created with biorender.com.](images/supplement/Figure-S1.png){#fig:FigS1}
```

Also, we want to be careful with referencing specific colors in the figure so the legend is color-blind accessible. Not immediately sure how to rephrase that, though.

Reply:
For the figures to render, the entire legend has to be on one line, so I will make the changes above. For the colors - do you mean yes reference or no don't reference colors, or should we change the colors?

Reply:
actually, I am going to separate these out since we are no longer linking Supp figures.